apps/web/pages/api/book/event.ts

```diff
@@ -249,8 +255,24 @@ async function handler(req: NextApiRequest) {
 
   const eventType =
     !eventTypeId && !!eventTypeSlug ? getDefaultEvent(eventTypeSlug) : await getEventTypesFromDB(eventTypeId);
+
   if (!eventType) throw new HttpError({ statusCode: 404, message: "eventType.notFound" });
 
+  // Check if required custom inputs exist
+  if (eventType.customInputs) {
+    eventType.customInputs.forEach((event) => {
+      if (
+        event.required === true &&
+        (reqBody.customInputs.length === 0 ||
+          reqBody.customInputs.filter(
```

Comment:
Can reqBody.customInputs ever be empty? If so `.length` could crash

Reply:
i don't it would crash as this is also being used on line 509
```
  if (reqBody.customInputs.length > 0) {
    reqBody.customInputs.forEach(({ label, value }) => {
      customInputs[label] = value;
    });
  }
  ```